src/test/java/SetTest.java

```diff
@@ -0,0 +1,46 @@
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set.size() 메서드 활용 테스트")
+    void request1() {
+        int expectedSize = 3;
+        int actualSize = numbers.size();
+
+        Assertions.assertThat(actualSize).isEqualTo(expectedSize);
```

Comment:
```suggestion
        int expected = 3;
        int actual = numbers.size();

        assertThat(actualSize).isEqualTo(expectedSize);
```

`DisplayName` 이나 `테스크명`으로 충분한 의미가 전달된다면
`Size`이름을 붙이지 않아도 충분한 의미가 통할것 같아요.
그리고, `Assertions`는 붙이지 않아도 될것 같네요! 😄